pkg/operator/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -179,15 +175,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	// Watch Secrets and reconcile if we see one with our label.
 	err = c.Watch(
-		&source.Kind{Type: &corev1.Secret{}},
+		source.Kind(mgr.GetCache(), &corev1.Secret{}),
 		targetCredSecretMapFunc,
 		p)
 	if err != nil {
 		return err
 	}
 
 	// allCredRequestsMapFn simply looks up all CredentialsRequests and requests they be reconciled.
-	allCredRequestsMapFn := handler.EnqueueRequestsFromMapFunc(func(a client.Object) []reconcile.Request {
+	allCredRequestsMapFn := handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, a client.Object) []reconcile.Request {
 		log.Info("requeueing all CredentialsRequests")
 		crs := &minterv1.CredentialsRequestList{}
 		err := mgr.GetClient().List(context.TODO(), crs)
```

Comment:
Pass the context you got in the method signature here

Reply:
done in upcoming commit